test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+rm *.tar.gz
+rm ~/repos/thing.txt
+R CMD BUILD .
+cp *.tar.gz ~/Desktop
+pushd ~/Desktop
+    R CMD CHECK *.tar.gz --as-cran
+    cat ~/repos/thing.txt
```

Comment:
I think the test script and the DEBUG lines are assuming too much about someone's directory structure, i.e., that they have a `~/repos` directory and a `~/Desktop` directory. I think you should just put everything in `~` or explicitly create a temp directory in `~` and name it something that is unlikely to collide.

Reply:
makes sense! I'll change it